kafka_consumer/hatch.toml

```diff
@@ -3,7 +3,7 @@
 [[envs.default.matrix]]
 python = ["2.7", "3.8"]
 # Note: 0.9 is for testing legacy pre-0.10.2 implementation
-version = ["0.9"]
```

Comment:
Sorry if this is a stupid question, but can't we already test the `latest` version with `ddev test --latest kafka_consumer`?

Reply:
ah good point, I'll take `latest` out and pin the version to the latest version